MPF.UI/Themes/CustomTheme.cs

```diff
@@ -0,0 +1,276 @@
+            this.ComboBox_MouseOver_Editable_Button_Background = backgroundBrush;
+            this.ComboBox_Pressed_Background = backgroundBrush;
+            this.ComboBox_Pressed_Editable_Background = backgroundBrush;
+            this.ComboBox_Pressed_Editable_Button_Background = backgroundBrush;
+            this.ComboBox_Static_Background = backgroundBrush;
+            this.ComboBox_Static_Editable_Background = backgroundBrush;
+            this.ComboBox_Static_Editable_Button_Background = backgroundBrush;
+
+            // Handle CustomMessageBox-specific resources
+            this.CustomMessageBox_Static_Background = backgroundBrush;
+
+            // Handle MenuItem-specific resources
+            this.MenuItem_SubMenu_Background = backgroundBrush;
+            this.MenuItem_SubMenu_Border = Brushes.DarkGray;
+
+            // Handle ProgressBar-specific resources
+            this.ProgressBar_Background = backgroundBrush;
+
+            // Handle ScrollViewer-specific resources
+            this.ScrollViewer_ScrollBar_Background = backgroundBrush;
+
+            // Handle TabItem-specific resources
+            this.TabItem_Selected_Background = backgroundBrush;
+            this.TabItem_Static_Background = backgroundBrush;
+            this.TabItem_Static_Border = Brushes.DarkGray;
+
+            // Handle TextBox-specific resources
+            this.TextBox_Static_Background = backgroundBrush;
+        }
+
+        /// <summary>
+        /// Check whether a string represents a valid hex color
+        /// </summary>
+        public static bool IsHexColor(string? color)
+        {
+            if (string.IsNullOrWhiteSpace(color))
+                return false;
+
+            if (color!.Length == 7 && color[0] == '#')
+                color = color.Substring(1);
+
+            if (color.Length != 6)
+                return false;
+
+            for (int i = 0; i < color.Length; i++)
+            {
+                switch (color[i])
+                {
+                    case '0':
+                    case '1':
+                    case '2':
+                    case '3':
+                    case '4':
+                    case '5':
+                    case '6':
+                    case '7':
+                    case '8':
+                    case '9':
+                    case 'A':
+                    case 'B':
+                    case 'C':
+                    case 'D':
+                    case 'E':
+                    case 'F':
+                    case 'a':
+                    case 'b':
+                    case 'c':
+                    case 'd':
+                    case 'e':
+                    case 'f':
+                        continue;
+                    default:
+                        break;
+                }
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Converts string hex to int hex
+        /// </summary>
+        private static byte HexToHex(string? hex)
```

Comment:
Please convert each of the switches to a call to a helper method that gets the value and just have it multiply by 16 for the first one. You can also probably consolidate the switches in the new syntax:
```
return c switch
{ 
    '0' => 0,
    ...
    _ => 0,
};
```